Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/checkout.py

```diff
@@ -39,7 +39,7 @@ def parser(cls, parser, loggers=None):
         parser.add_argument(
             'argument', nargs=1,
             type=str, default=None,
-            help='String representation of a commit or branch to be normalized',
+            help='String representation of a commit, branch or pull request (of for pr-#) to apply locally',
```

Comment:
"of for" ?

Reply:
Not quite sure what happened there, updated just now.